scripts/testy_launch.py

```diff
@@ -54,28 +54,34 @@ def get_snapshot_status(snapshot_id):
     snapshot_status = result.stdout.strip()
     return snapshot_status
 
-def get_value_from_launch_template(launch_template_name, key):
-    result = local(f"aws ec2 describe-launch-templates \
-        --launch-template-name {launch_template_name} \
-        --query 'LaunchTemplates[*].Tags[?Key==`{key}`].Value[]' \
+def set_tag_for_resource(resource_id, key, value):
+    result = local(f"aws ec2 create-tags \
+        --resources {resource_id} \
+        --tags 'Key={key},Value={value}'", hide=True, warn=True)
+    if result.stderr:
+        raise Exit(result.stderr)
+
+def get_tag_value_from_resource(resource_id, key):
+    result = local(f"aws ec2 describe-tags \
+        --filter 'Name=resource-id,Values={resource_id}' \
+        --query 'Tags[?Key==`{key}`].Value' \
         --output text", hide=True, warn=True)
     if result.stderr:
         raise Exit(result.stderr)
     value = result.stdout.strip()
     if not value:
-        raise Exit(f"Error: Unable to retrieve '{key}' for '{launch_template_name}'")
+        raise Exit(f"Unable to retrieve value for key '{key} from resource '{resource_id}'")
```

Comment:
Missing closing ' for the key: `'{key}`

Reply:
Fixed in [375bf2c](https://github.com/wiredtiger/testy/pull/50/commits/375bf2c0fe2fcd7e8659328e89af52a307578bb6).